.php-cs-fixer.dist.php

```diff
@@ -2,10 +2,9 @@
 
 declare(strict_types=1);
 
-require_once __DIR__ . '/bin/dev/php-cs-fixer/vendor/autoload.php';
-
 $finder = PhpCsFixer\Finder::create()
 	->in(__DIR__)
+	->notPath('addons')
```

Comment:
Shouldn't this be `addon` ?

Reply:
Yes, of course. I've force-pushed my branch, including this change.